File: Web/RMA-Web-Shell/projects/clientcare/src/app/product-manager/models/product-option-allowance-type.ts
import { AllowanceTypeEnum } from 'projects/shared-models-lib/src/lib/enums/allowance-type-enum';
import { IndustryClassEnum } from 'projects/shared-models-lib/src/lib/enums/industry-class.enum';

export class ProductOptionAllowanceType {
  productOptionAllowanceTypeId: number;
  productOptionId?: number;
  allowanceType?: AllowanceTypeEnum;
  industryClass?: IndustryClassEnum;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Product/ProductOptionAllowanceType.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;


namespace RMA.Service.ClientCare.Contracts.Entities.Product
{
    public class ProductOptionAllowanceType
    {
        public int ProductOptionAllowanceTypeId { get; set; }
        public int? ProductOptionId { get; set; }
        public AllowanceTypeEnum? AllowanceType { get; set; }
        public IndustryClassEnum? IndustryClass { get; set; }
    }
}